<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCUUpgrade</class>
 <widget class="QWidget" name="MCUUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1696</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MCUUpgrade {
background: #FFFFFF;
border-radius: 10px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="grpBox_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid rgb(200, 200, 200);
    font-size: 24px;
    font-family: &quot;PingFang SC&quot;;
    border-radius: 5px;
    margin-top: 2ex;
	font-weight: 500;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    left: 0px;
    margin-left: 0px;
    padding: 0 1px;
	color: #666666;
}</string>
     </property>
     <property name="title">
      <string>上位机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QPushButton" name="pushButton_app_upgrade">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>620</y>
        <width>180</width>
        <height>62</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>PingFang SC</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #437AFF;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>上位机升级</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpBox_9">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid rgb(200, 200, 200);
    font-size: 24px;
    font-family: &quot;PingFang SC&quot;;
    border-radius: 5px;
    margin-top: 2ex;
	font-weight: 500;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    left: 0px;
    margin-left: 0px;
    padding: 0 1px;
	color: #666666;
}</string>
     </property>
     <property name="title">
      <string>下位机</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 93px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #666666;
line-height: 35px;</string>
      </property>
      <property name="text">
       <string>升级文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_path">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>600</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 688px;
height: 60px;
background: #FFFFFF;
border-radius: 10px;
border: 1px solid rgba(170,189,212,0.6);

padding-left: 10px;

font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
      </property>
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QComboBox" name="com_device">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>600</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid #DCDEE0;
    border-radius: 6px;
    font-family: &quot;PingFang SC Medium&quot;;
    font-size: 24px;
    padding-left: 10px;
    background-color: rgb(250, 251, 252);
	font-weight: 500;
    color: #000000;
}

QComboBox::drop-down {
    margin: 0px;
    padding-right: 20px;
    border: 0px;
}

QComboBox::down-arrow {
    image: url(:/static/PHOTOS/combo_up.png);
}

QComboBox::down-arrow:on {
    image: url(:/static/PHOTOS/combo_down.png);
}

QListView {
    margin: 0px;
    padding: 0px;
    border: 0px;
    border-radius: 10px;
    outline: 0px;
    font-size: 24px;
    color: #333333;
    background-color: rgb(250, 251, 252);
}

QListView::item {
    margin: 0px;
    padding: 10px;
    border: 0px;
    border-radius: 0px;
    height: 60px;
}

QListView::item:hover {
    color: #333333;
}

QListView::item:selected {
    color: rgb(31, 112, 227);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 93px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #666666;
line-height: 35px;</string>
      </property>
      <property name="text">
       <string>升级设备</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_upgrade">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>620</y>
        <width>180</width>
        <height>62</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>PingFang SC</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #437AFF;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>下位机升级</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_upgrade_remote">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>620</y>
        <width>180</width>
        <height>62</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>PingFang SC</family>
        <pointsize>-1</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #437AFF;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>远程升级</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
